app/src/components/ProposalItem.tsx

```diff
@@ -38,11 +38,14 @@ class ProposalItem extends React.Component<ProposalItemProps, ProposalItemState>
       .call()
       .then((proposalData: any): void => {
         const proposal = {
-          title: proposalData[2],
-          medium: proposalData[3],
-          edition: proposalData[4],
-          created: proposalData[5],
-          metaUri: proposalData[6],
+          title: proposalData[1],
+          artistName: proposalData[2],
+          artistNationality: proposalData[3],
+          artistBirthYear: proposalData[4],
+          createdDate: proposalData[5],
```

Comment:
It is here where you give the property the name createdDate, you might be getting confused with the properties in Registry which have been given different names. Either switch the names to be the same for consistency or use the correct name in this case